cdc/owner/changefeed.go

```diff
@@ -262,10 +260,8 @@ func (c *changefeed) preflightCheck(captures map[model.CaptureID]*model.CaptureI
 			if status == nil {
 				status = &model.ChangeFeedStatus{
 					// the changefeed status is nil when the changefeed is just created.
-					// the txn in start ts is not replicated at that time,
-					// so the checkpoint ts and resolved ts should less than start ts.
-					ResolvedTs:   c.state.Info.StartTs - 1,
-					CheckpointTs: c.state.Info.StartTs - 1,
+					ResolvedTs:   c.state.Info.StartTs,
+					CheckpointTs: c.state.Info.StartTs,
```

Comment:
Does it mean TiCDC excludes the txn at the start-ts? @amyangfei

Reply:
No, txn with `start-ts` is included, in a reentrant way

Reply:
No, txn with start-ts is included, in a reentrant way

Reply:
Does this PR change TiCDC behavior?

Reply:
This logic is the same as the old owner.